### PR TITLE
coreos-layering-configuring: Remove cliwrap from the example

### PR DESCRIPTION
**Pull request by jmarrero**
Doc update for OCPBUGS-33483

Version(s):
4.16 and up

Issue:
OCPBUGS-33483

Link to docs preview:

https://77066--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/coreos-layering.html
QE review:
- [x] QE has approved this change.

Additional information:

This is required along with changes to rpm-ostree for OCP 4.16 and up, if the cliwrap is left in the docs users will encounter: https://issues.redhat.com/browse/OCPBUGS-30149

### CODE REVIEW

**Review by jlebon — APPROVED** [review]
LGTM, though I guess let's wait until OCPBUGS-33483 is VERIFIED to merge it.

**Comment by jmarrero** [comment]
OCPBUGS-33483 has been VERIFIED.

**Review by jldohmann — APPROVED** [review]
thanks for opening this Joseph! LGTM

**Review by HuijingHei — APPROVED** [review]
/lgtm

**Pull request merged**
